tutorials/3d/fps_tutorial/part_one.rst

```diff
@@ -8,15 +8,37 @@ Tutorial introduction
 
 .. image:: img/FinishedTutorialPicture.png
 
+.. error:: TODO: redo this image!
+
 This tutorial series will show you how to make a single player FPS game.
 
-Throughout the course of these tutorials, we will cover how:
+Throughout the course of this tutorial series, we will cover how:
 
-- To make a first person character, with sprinting and a flash light.
+- To make a first person character that can:
```

Comment:
Putting some of the bullets below right after this line feels weird
```
To make a first person character that can: Move around the environment      (okay)
To make a first person character that can: That can jump into the air ...   (redundant)
To make a first person character that can: That has a flash light we ...    (hmm)
and so on
```

I'm not sure if I explain clearly or even correctly, but maybe you could do something with it. :D